### PR TITLE
Jenkins Build

### PR DESCRIPTION
**Pull request by ffalcn**
Initial addition of Jenkinsfile and helm files. I haven't been able to test this in Jenkins yet (having some permissions problems) but wanted to get the initial review started. I based this off of the v1.11.1 branch as it seemed the most up to date.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      stage('approve to production') {
+        approval {
+          message = 'Deploy to production?'
+          okText = 'Deploy'
+          timeoutTime = 10
+        }
+      }
+
+      stage('deploy to production') {
+        standardRelease(config, version_tag) {}
```
Comment (campbelldgunn) [inline Jenkinsfile]: @ffalcn Does it need to deploy to production? The fact that it is published to ACR as an official image means it is ready for use by production pipelines.

File: charts/k8s-kubectl/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta2
```
Comment (campbelldgunn) [inline charts/k8s-kubectl/templates/deployment.yaml]: @ffalcn I don't think you need the helm charts. As this is an image that is consumed by other pipelines to build and deploy from their respective pipelines?

**Review by campbelldgunn — CHANGES_REQUESTED** [review]
I don't you need the chart files it is more about getting the image up to date that is consumed by other pipelines.

File: Jenkinsfile (hunk 1 above)
Comment (ffalcn) [reply Jenkinsfile]: I wasn't sure if it did or not, but left it in as it followed the example. I will remove it.

File: charts/k8s-kubectl/templates/deployment.yaml (hunk 2 above)
Comment (ffalcn) [reply charts/k8s-kubectl/templates/deployment.yaml]: That makes sense, I will remove the charts.

**Review by campbelldgunn — COMMENTED** [review]
How did you validate this? I did not see it in the dev/sandbox of Jenkins.

**Comment by ffalcn** [comment]
So this hasn't been validated yet, I don't yet have access to the Jenkins dev environment. Jake was working on getting the permissions for me yesterday, but there seems to have been a problem getting the changes through. I am checking with him now to see if he has any kind of update.

**Comment by ffalcn** [comment]
In checking with Jake he hasn't had any luck getting the permissions sorted out and suggested we check with Bob when he gets back.

**Comment by ffalcn** [comment]
The branch and pull request builds now run successfully. I examined the logs from the build and didn't see any issues, is there anything else you'd like me to add to this?

**Comment by campbelldgunn** [comment]
@ffalcn Can you make sure that anything in the docker image that is referencing the public is now referencing internal. I think there are few references in the docker file.

File: README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@
 [![](https://images.microbadger.com/badges/commit/campbelldgunn/k8s-kubectl.svg)](http://microbadger.com/images/campbelldgunn/k8s-kubectl "Get your own commit badge on microbadger.com")
```
Comment (campbelldgunn) [inline README.md]: We can remove the microbadger badges out as well.

**Review by campbelldgunn — CHANGES_REQUESTED** [review]

**Review by campbelldgunn — APPROVED** [review]
LGTM

**Pull request merged**